driver/main_args.ml

```diff
@@ -597,7 +597,22 @@ let mk_color f =
   \      never   disable colors\n\
   \    The default setting is 'auto', and the current heuristic\n\
   \    checks that the TERM environment variable exists and is\n\
-  \    not empty or \"dumb\", and that isatty(stderr) holds."
+  \    not empty or \"dumb\", and that isatty(stderr) holds.\n\
+  \  If the option is not specified, these setting can alternatively\n\
+  \  be set through the OCAML_COLOR environment variable."
+;;
+
+let mk_error_style f =
+  "-error-style", Arg.Symbol (["contextual"; "classic"], f),
+  Printf.sprintf
+    "  Control the way error messages and warnings are printed\n\
+    \    The following settings are supported:\n\
+    \      classic     only print the error and its location\n\
```

Comment:
That will only talk to old timers 👴. Here's a brain dump: `short`, `brief`, `minimal`, `location` ?

Reply:
(I'm not fond of "location" but the other suggestions seem fine.)

Reply:
Good point. I went with `short`.